packages/react-components/src/molecules/LabeledDateField.tsx

```diff
@@ -31,11 +47,14 @@ const LabeledDateField: React.FC<LabeledDateFieldProps> = ({
           id={id}
           value={value ? formatISO(value, { representation: 'date' }) : ''}
           onChange={(newDate) => onChange(parseISO(newDate))}
+          max={max ? format(max, 'yyyy-MM-dd') : undefined}
         />
       )}
     >
       <Paragraph>
         <strong>{title}</strong>
+        <span css={subtitleStyles}>{subtitle}</span> <br />
```

Comment:
Since the fields are added, are there no regressions in other usages of the component?

Reply:
This is the first time we are using this component